lib/jira/util/index.js

```diff
@@ -1,6 +1,19 @@
-const { markdown } = require('markdown');
-
 module.exports = function (jiraClient) {
+  const checkForReferenceText = (text) => {
+    const splitTextByNewLine = text.split('\n');
+
+    return splitTextByNewLine
+      // reference text links should have 2 parts only
+      .filter((newLine) => newLine.split(' ').length === 2)
+      .filter((text) => {
+        const hasSquareBrackets = text.charAt(0) === '[' && text.includes(']:');
+        const hasUrl = text.includes('http://') || text.includes('https://');
```

Comment:
Does a reference link have to have an HTTP or HTTPS URL? Could it not also have a relative URL starting with `/` or an URL with a completely different protocol?